integration_tests/requirements.txt

```diff
@@ -16,4 +16,5 @@ sre_yield
 pandas
 pyarrow
 pytest-xdist >= 2.0.0
-findspark
\ No newline at end of file
+findspark
+fastparquet
```

Comment:
Can we update the readme to include that we need fastparquet installed? Are there a specific version we want to test with? We have seen errors in some versions that are not in others.

Reply:
Yep. This is all a bit creaky at the moment. 0.7.1 does not play nicely with Spark versions preceding 3.4.x, it seems.